### DISCUSSION TITLE
Language bar has been disable but comes back on reboot

### DISCUSSION THREAD

**Discussion by caccamo88** [valinet/ExplorerPatcher#1753] [Q&A]
hi guys
wondering if Language bar* comes back on reboot is a bug of EP.
It has been correctly disabled but it stay disabled until next reboot.
Thanks

*To make it appears you must have at least 2 language installed

![immagine](https://github.com/valinet/ExplorerPatcher/assets/12472438/30965d09-b805-4da8-a76c-9b0cdfebfc87)

![immagine](https://github.com/valinet/ExplorerPatcher/assets/12472438/afbe7f99-27c3-4193-89c6-97efd142857f)

**Comment by pyrates999** [comment]
Please post windows version and build.
Please post explorer patcher version.

**Comment by winklejude** [comment]
the same problem

Windows 11 pro 22H2  22621.1992
ep  22621.1992.56.1

two different pc , all the same version

**Comment by pyrates999** [comment]
Try the following:

1. uninstall explorer patcher
2. delete the registry key: "Computer\HKEY_CURRENT_USER\Software\ExplorerPatcher"
3. reboot
4. install the latest explorer patcher
5. verify you get a notification that symbols have been downloaded
6. restart windows explorer using either the task manager or in the properties window of explorer patcher
7. reboot

Then apply your settings again in explorer patcher to see if it fixes it.

**Comment by caccamo88** [comment]
thanks but no way.
Have found a workaround

![immagine](https://github.com/valinet/ExplorerPatcher/assets/12472438/63742d4c-b6c6-4109-aa77-6253c09cbcab)

**Comment by pyrates999** [comment]
Can you list the steps you did for the work around?

**Comment by caccamo88** [comment]
simple: need to make appears, shortcut, to old (windows 10) task bar icon management and turn off "keybor input indicator". 
The problem is how to make appear it or at least me cannot find in windows 11.

But the trick is done thanks to EP setting "customize system icons in the notification area"

**Reply by winklejude** [reply]
It seems to work.
But if you change the first language settings
it will be reset
